exercises/conversions/from_into.rs

```diff
@@ -70,4 +70,10 @@ mod tests {
         assert_eq!(p.name, "Mark");
         assert_eq!(p.age, 20);
     }
+    #[test]
+    fn test_bad_age() {
+        let p = Person::from("Mark,twenty");
```

Comment:
A comment like in the tests above would be nice here!

Reply:
Done